### PR TITLE
feat: support gif

### PR DESCRIPTION
**Pull request by daniel-vang**

### CODE REVIEW

**Comment by daniel-vang** [comment]
Per testing, `{ animated: true }` doesn't impact non-animated format like png/jpg.

File: src/GetOrCreateImage/GetOrCreateImage.js [hunk 1]
```diff
@@ -46,8 +46,8 @@ const GetOrCreateImage = async event => {
 
       // Required try/catch because Sharp.catch() doesn't seem to actually catch anything. 
       try {
-        resizedImage = Sharp(imageObj.Body)
-          .resize(width, height)
+        resizedImage = Sharp(imageObj.Body, { animated: true })
+          .resize(width, height, {fit: 'inside'})
```
Comment (HoraceShmorace) [inline src/GetOrCreateImage/GetOrCreateImage.js:50]: Have you confirmed that `{fit: 'inside'}` doesn't have side effects for other formats?

**Comment by HoraceShmorace** [comment]
This repo doesn't use the Angular convention for Git commit messages.

**Comment by daniel-vang** [comment]
will create a more clean pr later.

**Pull request closed without merging**